java/example/calypso/android/omapi/src/main/kotlin/org/eclipse/keyple/example/calypso/android/omapi/activity/CoreExamplesActivity.kt

```diff
@@ -73,7 +74,7 @@ class CoreExamplesActivity : ExamplesActivity() {
                     addHeaderEvent("Starting explicitAidSelection with $poAid on Reader ${it.key}")
 
                     val seSelector = SeSelector.builder()
-                            .seProtocol(SeCommonProtocols.PROTOCOL_ISO7816_3)
+                            .seProtocol(ContactsCardCommonProtocols.ISO_7816_3.name)
```

Comment:
For the OMAPI selector example, I would recommend to not set the Card Protocol, especially because finally all the CardSelector are defined with the same protocol PROTOCOL_ISO7816_3

Reply:
Yes, this must be removed.